src/shared/analytics/shared/mixpanel-data-helpers.ts

```diff
@@ -157,7 +167,8 @@ export async function getBaseMixpanelParams(account: Account) {
     num_zerion_wallets: ownedAddressesCount,
     num_connected_wallets: 0,
     num_wallet_groups: groups?.length ?? 0,
-    total_balance: portfolioStats?.totalValue ?? 0,
+    total_balance: portfolioStats?.portfolio?.total_value ?? 0,
+    ...getChainBreakdown(portfolioStats?.portfolio ?? null),
```

Comment:
Do we need this breakdown in user_properties?

Reply:
Unfortunately :)